utils/utils.tsx

```diff
@@ -1157,8 +1165,8 @@ const UserStatusesWeight = {
 /**
  * Sort users by status then by display name, respecting the TeammateNameDisplay configuration setting
  */
-export function sortUsersByStatusAndDisplayName(users, statusesByUserId, teammateNameDisplay) {
-    function compareUsers(a, b) {
+export function sortUsersByStatusAndDisplayName(users: UserProfile[], statusesByUserId: Record<UserProfile['id'], keyof typeof UserStatusesWeight>, teammateNameDisplay: string) {
```

Comment:
This whole function and `UserStatusesWeight` can actually be removed. I can't find it being used anywhere outside of its own tests